NodeDebug/entry_point.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# IMPORTANT: Change this file only in directory NodeDebug!
+
```

Comment:
File added to handle the `entry_point.sh` in the same way it is done for the StandaloneDebug images. Things get changed here and the file is copied to the appropriate folders.